woodwork/statistics_utils/_get_describe_dict.py

```diff
@@ -62,6 +62,7 @@ def _get_describe_dict(
         "category": ["count", "nunique"],
         "numeric": ["count", "max", "min", "nunique", "mean", "std"],
         Datetime: ["count", "max", "min", "nunique", "mean"],
+        Unknown: ["nunique"],
```

Comment:
Need to add `"count "` or Unknown columns will no longer get count calculated for them (see line 122 where all other columns get `"count"` calculated for them

Reply:
added